elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -317,6 +317,7 @@ $variables: map-merge($variables, $nav-item-variables);
     color:            #{pfe-local($cssvar: Color, $region: tray)};
     //mobile tray padding
     padding: #{pfe-local($cssvar: Padding, $region: tray)};
+    // padding: 64px;
```

Comment:
Is this commented out code still needed?

Reply:
I should've removed this. I did in the most recent commit.